src/screens/HomeScreen.js

```diff
@@ -146,104 +231,65 @@ export const HomeScreen = ({ navigation, route }) => {
     }
   };
 
-  return (
-    <SafeAreaViewFlex>
-      <ScrollView
-        refreshControl={
-          <RefreshControl
-            refreshing={refreshing}
-            onRefresh={refresh}
-            colors={[colors.accent]}
-            tintColor={colors.accent}
-          />
-        }
-      >
-        <ConnectedImagesCarousel
-          alternateAspectRatio
-          navigation={navigation}
-          publicJsonFile="homeCarousel"
-          refreshTimeKey="publicJsonFile-homeCarousel"
+  if (isShow && !!categoriesNews) {
```

Comment:
`categoriesNews` is an array with objects. even ifit has no objects and is an empty array, this check would give `true`. as we `map` over the data in the condition, i suggest to check for `.length` here.